applications/polkarun.md

```diff
@@ -0,0 +1,229 @@
+# polka.run - PolkaVM Disassembler
+
+- **Team Name:** [Rotko Networks](https://rotko.net)
+- **Payment Address:** 1Qrotkokp6taAeLThuwgzR7Mu3YQonZohwrzixwGnrD1QDT (USDC on Asset Hub)
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 2
+
+## Project Overview
+
+### Overview
+
+polka.run aims to be a premier website offering web tools designed to demystify PolkaVM bytecode.
+The cornerstone of this suite is a disassembler GUI, enabling users to convert .polkavm binary
+files into human-readable formats.  
+
+### Key Features
+
+- Ability to read binary data in both HEX and ASCII formats.
+- A robust disassembler that translates binary data into RISC-V assembly instructions.
+- A user-friendly GUI, ensuring easy interaction and navigation for various skill levels.
```

Comment:
This is a contradiction. People that are going to read and make sense of some random RISC-V assembly code (which is always hard) are certainly capable of installing and using the `polkatool` command line App. That said, a WebApp can certainly be more user friendly and more pleasant to use for _some_ people (but it is a question of personal preference).